src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
I don't see any need to keep it. `originalEvent` is useful and should definitely stay around, but I can't think of any existing use cases for `originalEvent` in simulated events since they've been empty objects for the past four years.